pytest_replay.py

```diff
@@ -66,12 +66,12 @@ def append_test_to_script(self, nodeid):
 
 class TemplateBat:
     preamble = 'REM generated by pytest-replay\npytest %* ^\n'
-    test = '  {nodeid} ^\n'
+    test = '  "{nodeid}" ^\n'
```

Comment:
I will suguest to add the escape character + new line (`^\n` and `\\\n`) at the start of the "test" member and removing it from the preamble.
A dangling escape at the end of file looks like and error. I don't know if the shells will cause an error due that.

Reply:
Done, thanks!